translationCoreApps/translationWords_Check_plugin
# VerseCheck

This PR should be up to date with Manny’s latest PR’s and Check Info pane. 

**This PR addresses:**
Integration of VerseCheck component/module Abstraction.

**To test this PR**
Checkout VerseCheck repo to tC_apps folder, and run the app and see how it behaves. Please document here any other modifications you need to do to get the tool up and running with VerseCheck.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/translationwords_check_plugin/46)
